msal/src/main/java/com/microsoft/identity/client/SingleAccountPublicClientApplication.java

```diff
@@ -432,6 +517,8 @@ private MultiTenantAccount getAccountFromICacheRecordList(@NonNull List<ICacheRe
     @Override
     public void acquireTokenSilentAsync(@NonNull final String[] scopes,
                                         @NonNull final AuthenticationCallback callback) {
+
+        //TODO: Add null check for getPeristedAccount and throw exception if account not found
```

Comment:
Something to resolve in this PR?

Reply:
I just did.  Thanks for reminding me.  I noticed this was missing when refactoring for sync.